stable/mariadb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mariadb
-version: 5.11.1
+version: 5.11.2
```

Comment:
```suggestion
version: 5.12.0
```

Reply:
Oh - I need a signed commit - let me go create that...